index.html

```diff
@@ -3074,94 +3078,76 @@ <h3>WoT Binding Templates</h3>
         <figcaption>From Binding Templates to Protocol Bindings</figcaption>
       </figure>
       <p> <a href="#fig-binding-templates"></a> shows how <a>Binding
-          Templates</a> are applied. A <a>WoT Binding Template</a>
-        is created only once for each <a>IoT Platform</a> and
-        can then be reused in all <a>TDs</a> for devices of that
-        platform. The <a>Consumer</a> that is processing a <a>TD</a> must implement
-        the required <a>Protocol Binding</a> by including a
-        corresponding protocol stack and by configuring the
-        stack (or its messages) according to the information
-        given in the <a>TD</a>.
+          Templates</a> are used. Based on the protocol, media type or platform binding, 
+          a <a>TD</a> is created. The <a>Consumer</a> that is processing a <a>TD</a> 
+          implements the required <a>Binding Template</a> by including a corresponding 
+          protocol stack, media type encoder/decoder or platform stack and by configuring
+           the stack (or its messages) according to the information given in the <a>TD</a>.
       </p>
+
       <p>
-        The communication metadata of
-        <a>Protocol Bindings</a> spans five dimensions:
+        The <a>Binding Templates</a> spans three dimensions:
       </p>
       <ul>
         <li>
-          <b>IoT Platform:</b>
-          <p>
-            <a>IoT Platforms</a> often
-            introduce proprietary modifications at the
-            application layer such as platform-specific HTTP
-            header fields or CoAP options. Forms (see <a href="#sec-hypermedia-forms"></a>) may contain the
-            necessary information to apply these tweaks in
-            additional form fields defined for the
-            application-layer protocol used.
-          </p>
-        </li>
-        <li>
-          <b>Media Type:</b>
+          <b>Protocol</b>
           <p>
-            <a>IoT Platforms</a> often differ in the representation formats
-            (a.k.a. serializations) used for exchanging data.
-            The media type [[!RFC2046]] identifies these formats,
-            while media type parameters may specify them further.
-            Forms may contain the media type and optional parameters in
-            additional form fields such as a content type field known from HTTP,
-            which combines media type and its potential parameters
-            (e.g., <code>text/plain; charset=utf-8</code>).
-          </p>
-        </li>
-        <li>
-          <b>Transfer Protocol:</b>
-          <p>
-            The Web of Things uses the term <a>transfer protocol</a> for the underlying, standardized
+            The Web of Things uses the term <a>transport protocol</a> for the underlying, standardized
             application-layer protocol without application-specific options or <a>subprotocol</a> mechanisms.
             The URI scheme of the form submission target contains the information required to
```

Comment:
At this point the reader does not know what a "form submission target" is. Consider mentioning "Interaction Affordance and associated protocol endpoint URI".

Reply:
I did not change this actually but you are right, I will update

Reply:
Done